### PR TITLE
Update notes in vmware_guest.py - SysPrep Windows Guest

### PR DESCRIPTION
**Pull request by nmandy-git**
##### SUMMARY
Adding notes to be aware that sysprep is part of the deployment process when matching Windows guest

##### ISSUE TYPE
- Docs Pull Request
- Missing notes
+label: docsite_pr

##### COMPONENT NAME
vmware_guest.py

### CODE REVIEW

**Review by goneri — APPROVED** [review]

File: lib/ansible/modules/cloud/vmware/vmware_guest.py [hunk 1]
```diff
@@ -40,6 +40,7 @@
     - "Module may require additional privileges as well, which may be required for gathering facts - e.g. ESXi configurations."
     - Tested on vSphere 5.5, 6.0, 6.5 and 6.7
     - Use SCSI disks instead of IDE when you want to expand online disks by specifying a SCSI controller
+    - Use SysPrep for Windows VM (depends on 'guest_id' parameter match 'win') with PyVmomi
```
Comment (cigamit) [inline lib/ansible/modules/cloud/vmware/vmware_guest.py]: I am not sure the wording is 100% correct on it, "Use sysprep" makes it sound like you should sysprep any Windows Template beforehand, when actually its the opposite.  Maybe should be "Uses"?

Comment (Akasurde) [reply lib/ansible/modules/cloud/vmware/vmware_guest.py]: I agree with @cigamit. Could you please change wording a bit ? Thanks,.

**Review by Akasurde — CHANGES_REQUESTED** [review]
In-line comment.

File: lib/ansible/modules/cloud/vmware/vmware_guest.py (hunk 1 above)
Comment (nmandy-git) [reply lib/ansible/modules/cloud/vmware/vmware_guest.py]: I totally agree, as English is not my native language wording and typos mistakes happen sometimes :/
Thanks for your vigilance!
I'm not a github specialist and this is obviously my first public commit/PR. I hope I did it good.
Regards

Comment (cigamit) [reply lib/ansible/modules/cloud/vmware/vmware_guest.py]: Looks good to me.
shipit

**Comment by Akasurde** [comment]
@nmandy-git Thanks for the contribution.

**Comment by Akasurde** [comment]
@cigamit @goneri Thanks for the review.

**Pull request merged**